For `test_model_parallelism`, in order to pass `self.assertSetEqual(set(new_model.hf_device_map.values()), {0, 1})`, I need to set a higher ration (`0.5` --> `0.6`), but then I get errors for some models:

```bash
        position_embeddings = self.position_embedding(position_ids)
>       embeddings = inputs_embeds + position_embeddings
E       RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cuda:1!
```

It looks to me we  should keep the higher values but have a fix in the modeling file to ensure the 2 embeddings will be on the same device. Is this correct?

--- a/tests/models/longformer/test_modeling_longformer.py
+++ b/tests/models/longformer/test_modeling_longformer.py
@@ -326,6 +326,9 @@ class LongformerModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCa
         else {}
     )
 
+    # Need to use `0.6` instead of `0.5` for `test_disk_offload`
+    model_split_percents = [0.6, 0.7, 0.9]
+
     # TODO: Fix the failed tests
     def is_pipeline_test_to_skip(
         self, pipeline_test_casse_name, config_class, model_architecture, tokenizer_name, processor_name

